hack/operatorhub/templates/csv.tpl

```diff
@@ -141,8 +141,8 @@ spec:
       name: {{ .Name }}
       version: {{ .Version }}
     {{- end }}
-  description: 'Elastic Cloud on Kubernetes automates the deployment, provisioning,
-    management, and orchestration of Elasticsearch, Kibana, APM Server, Beats, and
+  description: 'Elastic Cloud on Kubernetes (ECK) automates the deployment, provisioning,
+    management, and orchestration of Elasticsearch, Kibana, APM Server, Beats, and 
     Enterprise Search on Kubernetes.
 
```

Comment:
The name will most likely be displayed next to the description in most cases. So, Instead of repeating it, how about:

> Elastic Cloud on Kubernetes (ECK) is the official operator by Elastic for automating the deployment, provisioning,
    management, and orchestration of Elasticsearch, Kibana, APM Server, Beats, and 
    Enterprise Search on Kubernetes.

Reply:
Sounds good.